src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+
+                printLine();
+                System.out.println("\tGot it. I've added this task:");
+                System.out.println("\t" + listItems[pointer].toString());
+                System.out.println("\tNow you have " + (pointer + 1) + " tasks in the list.");
+                printLine();
+                pointer++;
+            }
+            // add to do task
+            else if (userInput.startsWith("todo")) {
+                try{
+                    String todoDescription = userInput.substring(5);
+                    CheckEmpty(todoDescription);
+                    listItems[pointer] = new Todo(todoDescription);
+                    printLine();
+                    System.out.println("\tGot it. I've added this task:");
+                    System.out.println("\t" + listItems[pointer].toString());
+                    System.out.println("\tNow you have " + (pointer + 1) + " tasks in the list.");
+                    printLine();
+                    pointer++;
+                }
+                catch (StringIndexOutOfBoundsException e) {
+                    printLine();
+                    System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    printLine();
+                    }
+            }
+
+          /*   else{
+                printLine();
+                listItems = storeUserList (listItems, userInput);
+                printLine();
+                } */
+        }
+    }
+
+    //store items in list
+    public static Task[] storeUserList(Task[] listItems, String input) {
+        int counter = 0;
+            while(true){
+                if(listItems[counter] == null){
+                    listItems[counter] = new Task(input);
+                    System.out.println ("\tadded: " + input);
+                    break;
+                }
+                counter++;
+            }
+        return listItems;
+    }
+
+        //print items in list
+    public static void printUserList(Task[] listItems){
+        if(listItems[0] == null){
+            System.out.println("List is empty");
+            return;
+        }
+
+        printLine();
+
+        for (int i = 1 ; i <= listItems.length ; i++){
+            System.out.println("\t" + i + ". " + listItems[i-1].toString());
+                if(listItems[i] == null){
+                    printLine();
+                    return;
+                }
+            }
+        }
+
+
+
+    public static Task[] doneMessage(Task[] listItems, String userInput)   {
+
+       try {
+           int filteredNumber = Integer.parseInt(userInput.substring(5));
+           listItems[filteredNumber - 1].markDone();
+
+           printLine();
+           System.out.println("\tNice! I've marked this task as done:");
+           System.out.println("\t" + listItems[filteredNumber - 1].toString());
+           printLine();
+
+       }
+        catch (NumberFormatException e){
```

Comment:
Cross check with the coding standard to see if your placement of `{` complies with it.